corehq/apps/hqwebapp/static/hqwebapp/js/alert_user.js

```diff
@@ -38,6 +38,9 @@ hqDefine("hqwebapp/js/alert_user", function() {
                 alert.alert_class(alert.alert_class() + ' ' + tags);
             }
         }
+
+        // Jump to top of page so user sees alert
+        $("html").scrollTop(0);
```

Comment:
I wish we had hovering alerts :(... 
Having the browser scroll so that I can't see what I just did is quite annoying and adds friction to making changes like this (e.g. I moved this form, but I have no feedback on what the current state is now, because it is out of the viewport. If the top of the screen is already visible, then now my browser just randomly scrolled up a few lines)